core/services/vrf/listener_v2.go

```diff
@@ -53,6 +54,15 @@ const (
 	// BatchFulfillmentIterationGasCost is the cost of a single iteration of the batch coordinator's
 	// loop. This is used to determine the gas allowance for a batch fulfillment call.
 	BatchFulfillmentIterationGasCost = 52_000
+
+	// backoffInitialDelay is the initial delay to wait before retrying a failed request.
+	backoffInitialDelay = time.Minute
+
+	// backoffMaxDelay is the maximum amount of time to wait between retries.
+	backoffMaxDelay = time.Hour
+
+	// backoffFactor is the factor by which to increase the delay each time a request fails.
+	backoffFactor = 1.2
```

Comment:
Do we want to make these job-level params? Or are these constants good enough for all chains / use cases?